src/main/java/com/github/nicholasmoser/mot/GNTAnimation.java

```diff
@@ -122,7 +134,7 @@ public byte[] getBytes() throws IOException {
             if ((lastSize*8) % 16 != 0){
               alignment = 16 - (lastSize*8) % 16;
             }
-            int timeValueOffset = currentOffset + (lastSize * 8);
+            // int timeValueOffset = currentOffset + (lastSize * 8);
```

Comment:
This line didn't appear to be used so I commented it out

Reply:
Classical me, make a variable to make the code more readable, skip using the variable to redo the calculation just below!